api/producer/updateDocumentReference/src/v1/handler.py

```diff
@@ -77,13 +77,36 @@ def document_pointer_exists(
     )
 
 
+def _sort_key(key):
+
+    if type(key) == list:
+        return sorted(key)
```

Comment:
does not work for list of dict, or a list of list, etc, i'd suggest:

```suggestion
        return sorted(key, key=_sort_key)
```